card.js

```diff
@@ -0,0 +1,27 @@
+export function createCharacterCard(datatvshow) {
+  const newCard = document.createElement("li");
+  ul.append(newCard);
+  newCard.classList.add("card-container");
+  newCard.innerHTML = `
+  <li class="card">
```

Comment:
the newCard already the li element itself. You should not have an li element again within the li element

Reply:
Right thx